### ISSUE TITLE
protease  activity vs. peptidase activity

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12794]
PomBase user:
On a different note, I am trying to find all genes encoding proteases. 
But there doesn’t seem to be a GO term that holds all proteases. Any 
suggestions?

Me:
it has the narrow synonym "protease activity", I'm not sure why. To me 
it seems exact. I will follow up on this too.

Does anyone know why "protease" is narrow?

Val

**Comment by ukemi** [comment]
I suspect it is because peptidase activity can act on any peptide bond and protease activity is specific to proteins. ??

**Comment by deustp01** [comment]
GO:0070011 peptidase activity, acting on L-amino acid peptides "Catalysis of the hydrolysis of peptide bonds formed between L-amino acids."

Many proteases have good activity as well as esterases, and (I guess) most if not all can be made to act on peptide bonds found in structures that are not proteins. And for many, the true substrate range in vivo may not be clear even if a well-known major in vivo substrate is a protein or group of proteins.

That, anyway, is how I remember the reasoning that went into the cleanup of activity terms to eliminate terms that referred to gene products and to re-organize the survivors according to cleavage mechanism and cleavage site (exo- vs endo-peptidases)

**Comment by ukemi** [comment]
Thanks Peter. So to get all proteases, would you search on the above term or on peptidase activity? I think the wider net would be better, don't you?

**Comment by ValWood** [comment]
OK, so protease is narrower, but requires a "protein" as a substrate in an extension.
An annotation comment to this effect would  probably be useful...

**Comment by ValWood** [comment]
Although it is a bit weird. A peptidase should be 
"an enzyme which breaks down peptides into amino acids"
  
Protease seems to be the broader term
A protease (also called a peptidase or proteinase) is any enzyme that performs proteolysis, that is, begins protein catabolism by hydrolysis of the peptide bonds that link amino acids together in a polypeptide chain.

and should probably be the primary term name.

**Comment by ValWood** [comment]
Shouldn't it be

protease
-peptidase
- proteinase

?

**Comment by deustp01** [comment]
David, you're right - the user should choose here depending whether the goal is to find everything that can attack peptide bonds or only the subset of those found in textbook proteins. The user looking broadly might also want to look for gene products annotated with esterase activity.
Val, my understanding (but advice from hard-core enzymologists is needed) that many so-called proteases can act on ester linkages (indeed the standard assays in vitro involve chromogenic ester substrates) so the issue is whether to organize the hierarchy to reflect the chemical abilities of the gene product (broad) or its likely major activity in vivo (assuming that can be reliably defined across a broad range of taxa).

**Comment by ValWood** [comment]
But we don't have a term for the set which act only on proteins, or do we?

**Comment by deustp01** [comment]
Don't know of one - are there gene products that could reliably get this activity annotation (there's a really strong NOT annotation wrapped up inside it)?

**Comment by ValWood** [comment]
all of these 
GO:0019783 ubiquitin-like protein-specific protease activity (currently 30 in this list for pombe  but some are missing, which makes this term not broad enough)
we would not expect these to act on peptides physiologically......

I don't think these need a NOT for "peptidase"

**Comment by ValWood** [comment]
also separase (just cleaves)

**Comment by ValWood** [comment]
Things which remove signal peptides...

**Comment by deustp01** [comment]
Okay - the protein-substrate-only class exists.

**Comment by ukemi** [comment]
So is there an ontology action here? I'm not really sure what to do?

**Comment by ValWood** [comment]
If there was a class separation in GO  we would annotate  things which do both (the proteasome?)
to both terms.

**Comment by deustp01** [comment]
Midori was involved in the reorganization that led to the current peptidase (protease) hierarchy. Could she be brought into the discussion of what to do next and how to do it without unintended consequences?

**Comment by ukemi** [comment]
That would be great.
@mah11 can you shed any light on this?

**Comment by ValWood** [comment]
So,
If I understood. There should be a grouping term for "peptidases" and "proteinases" (protease? the current peptidase term).
Then there should be  child terms for "peptidase" and "proteinase". 
However, this might be too much hassle, because, although some descendants  like
GO:0019783 ubiquitin-like protein-specific protease activity
would belong under proteinase, and GO:0070573 - metallodipeptidase would clearly belong under peptidase, a LOT of terms would need replicating under both.

Maybe  less upheaval to rename
"peptidase" to "protease" and make both "peptidase" and "proteinase" narrow synonyms.

@mah11  will be watching can you advise?

**Comment by mah11** [comment]
I don't recall every detail about the organization under "peptidase activity", beyond some of the high-level mechanism-based splits. What I do remember is that we worked closely with the experts at MEROPS (physically located at the Sanger Inst.), and they reviewed and approved the revised ontology structure that we proposed and then implemented.

As part of that collaboration, they gave us a clear and unequivocal message that the root-most term in GO MF should be named "peptidase activity", and that "protease activity" should be a synonym, not a GO term in its own right. I don't have as precise a memory about the scope of that synonym, but I don't remember anything that suggests they would object to it being narrow (and as I said, they reviewed everything, so I hope they would have noticed and squawked if a synonym scope looked wrong).

I think they also said there was historically an attempt to make a distinction between "protease" and "proteinase" but it didn't hold up, and nowadays there's no reason to make either anything other than a synonym of "peptidase activity". They certainly approved the removal of all the recognition sequence specificity (said they regretted letting so much into EC, and that GO MF would have the better approach post-revamp). I'm not sure I ever knew whether considerations of ester bonds or peptide vs. protein in vivo substrates formed part of their opinion.

In short, what GO MF has now reflects expert input and I doubt the experts we consulted would think it needs any changes.

**Comment by ValWood** [comment]
I guess what I don't understand, based on user feedback, is why protease is narrow (it seems broader to me, if I understood correctly protease includes proteinase and peptidase.?)

I guess the easiest is to ask Neil :) ndrawlings about the synonym scope.

**Comment by mah11** [comment]
What I understood from Neil Rawlings and Alan Barrett is that there's no way to reliably define a distinction between "peptidase", "protease", and "proteinase", and that "peptidase" is the preferred name.

The only possibly open question is whether NARROW is the best synonym scope. On that point I have neither relevant memory nor current preference.

**Comment by ValWood** [comment]
This implies "exact"
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1147080/pdf/biochemj00274-0301.pdf


but it also states "In the shorter term, it would be difficult to
recommend abandoning 'protease' completely until the
recent clarification of the meaning of 'peptidase' has had
time to gain acceptance. 'Proteinase' also can continue
to be used, although it is not entirely satisfactory,
because some might expect the word to apply to
exopeptidases that degrade proteins as well as to
endopeptidases. T

BUT, we need the GO term to include  "exopeptidases that degrade proteins" not only to peptidases???

**Comment by mah11** [comment]
I'm a bit confused by this last comment:

> BUT, we need the GO term to include "exopeptidases that degrade proteins" not only to peptidases???

The broadest GO term is "peptidase activity" (GO:0008233). It has "endopeptidase activity" and "exopeptidase activity" among its is_a descendants. What's the question?

**Comment by ukemi** [comment]
I don't know the exact history of this, but this makes sense to me. From the definition, peptidases can work on ANY peptide bond whether they are in peptides, proteins or anywhere else. Protease and proteinase were made narrow synonyms of peptidase because proteins are a subset of the things that contain peptide bonds. So I think peptidase activity is what you want. I agree with Midori, I don't really think there is an ontology change needed here.

**Comment by ValWood** [comment]
Everything I'm reading implies that protease/peptidase are exact (protease is not specific to proteins):
i.e wikipedia
https://en.wikipedia.org/wiki/Protease
and
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1147080/pdf/biochemj00274-0301.pdf

and clearly users  who work on proteases find it strange that  it is not easy to locate all proteases in GO....(we give less weight to narrow synonyms in our searches, so peptidase does not appear high in the list with a search on protease).

I'm not even sure that "peptidase" is the most popular usage. 

google 
peptidase 2,200,000
protease 14,000,000

For example peptidase/protease inhibitors are almost universally called "protease inhibitors"

I don't understand why this is confusing, unless I have misunderstood the difference between 
proteases and peptidases.

**Comment by mah11** [comment]
It was specifically this line that was confusing: "BUT, we need the GO term to include "exopeptidases that degrade proteins" not only to peptidases???" The rest is clear.

This is not a popularity contest. We are assured by the relevant experts that GO is correct in using "peptidase activity" as its primary term name.

I find David's argument for the narrow synonym scope persuasive.

**Comment by ukemi** [comment]
If you think that in common usage, they are exact synonyms, then I will change that restriction. Will that work?

**Comment by ValWood** [comment]
deleted the comment about exopeptidase, it applied to proteinase, not protease.
The rest still applies. We can wait for Merops to comment on the scope.

**Comment by ndrawlings** [comment]
Hi Val,

 

“Protease”, “proteinase” and “peptidase” are interchangeable, but “peptidase” is recommended by Enzyme Nomenclature.  There are some people who would not use “proteinase” unless the substrate being cleaved was a protein, so not a peptide, but in reality if a peptidase cleaves a protein it will cleave peptides as well.  On the other hand, there are peptidases that cleave peptides but can’t cleave proteins – oligopeptidases such as prolyl oligopeptidase, insulysin, neprilysin.  In MEROPS, we use “peptidase”.

**Comment by ValWood** [comment]
Thanks Neil,
So based on this, is it correct to say:
primary name  peptidase
exact synonym protease?
narrow synonym proteinase?

**Comment by ukemi** [comment]
Since not all peptidases can work on all proteins, this makes sense. I changed the synonym scope to reflect the above.

**Comment by ValWood** [comment]
Happy about that. That solves the immediate issue where users are not finding the term "protease".  I would not have bothered so much, but many users are focussing on proteins of unknown biological role (we include anything which has only "proteolysis" in this list because  a protease can act on so many varied processes).  Not having "protease" as an exact synonym of anything was problematic.
